### PR TITLE
chore: add license checker during linting phase

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ to be OSS complaint

### CODE REVIEW

**Comment by vigneshshanmugam** [comment]
@jahtalab Rebased against master.

File: scripts/license-checker.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  'CC0-1.0',
+  'CC-BY-3.0',
+  'CC-BY-4.0',
+  'Unlicense'
+]
+
+/**
+ * Packages that must be ignored during license check
+ */
+const IGNORE_LIST = ['colors@0.6.2']
```
Comment (jahtalab) [inline scripts/license-checker.js]: Why is this necessary?

Comment (vigneshshanmugam) [reply scripts/license-checker.js]: They have a custom license which is already fixed in colors@1.0 so this is a workaround to avoid the check for older versions.

Comment (vigneshshanmugam) [reply scripts/license-checker.js]:
```
license: Custom: http://i.imgur.com/goJdO.png
```
